src/ReflectionProperty.php

```diff
@@ -98,7 +124,7 @@ public function getTypeNode()
     public function ___debugInfo()
     {
         return array(
-            'name'  => isset($this->propertyNode) ? $this->propertyNode->name : 'unknown',
+            'name'  => isset($this->propertyName) ? $this->propertyName : 'unknown',
```

Comment:
The `isset` won't get you desired result, because `$this->propertyName` is now always set (thanks to your changes) from mandatory constructor argument.

Reply:
I have to investigate further, but you're probably right. If I can simplify this, I will. 👍